src/config.template.cjs

```diff
@@ -94,7 +94,7 @@ module.exports = {
 	 * 		dataDirs: ["/path/here", "/other/path/here"],
 	 * 		dataDirs: ["C:\\My Data\\Downloads"]
 	 */
-	dataDirs: undefined,
+	dataDirs: [],
```

Comment:
I like this, I also think that since I've ran across some users thinking that because I included multiple examples, that each path needs it's own line of `dataDirs:` perhaps removing the three examples for just one would be better? Or maybe explicitly stating to only use one and that they are just multiple possible examples.

The only thing is whether we transform an empty array to null or undefined, I believe it is currently transformed to null.

Reply:
I like this change as well. I wonder if updating the example to this would make it more clear:

```ts
example:
    dataDirs: ["/path/here"], OR
    dataDirs: ["/path/here", "/other/path/here"], OR
    dataDirs: ["C:\\My Data\\Downloads"]
```

My only concern is whether people reading this remove the "OR" or keep it and break the config?